Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            
```

Comment:
Missing something here 😄

Reply:
what shall i write here? Messages prompting for name, surname, age and height/weight?

Reply:
Yes, as the name implies, demo that the method works.

Reply:
Fixed it and all the other problems, Should be fine now